instrumentation/packages.go

```diff
@@ -832,6 +833,18 @@ var packages = map[Package]PackageInfo{
 	PackageOS: {
 		TracedPackage: "os",
 	},
+	PackageMark3LabsMcpGo: {
+		TracedPackage: "github.com/mark3labs/mcp-go",
+		EnvVarPrefix:  "MCPGO",
+		naming: map[Component]componentNames{
+			ComponentServer: {
+				useDDServiceV0:     true,
+				buildServiceNameV0: staticName("mcp.server"),
```

Comment:
```suggestion
				buildServiceNameV0: staticName("mcp-server"),
```

(the dot notation is mostly used in operation names / resource names)